include/ignition/gui/qml/GzPose.qml

```diff
@@ -124,6 +126,19 @@ Item {
     }
   }
 
+  /**
+   * Used to create a plotting icon
+   */
+  Component {
+    id: gzPlotPose
+    GzPlot {
+      gzMimeData : { "text/plain" : (model === null) ? "" :
+          "Component," + model.entity + "," + model.typeId + "," +
+          model.dataType + "," + gzComponentInfo + "," + model.shortName
```

Comment:
This is `gz-sim` specific. We should just expose `gzMimeData` and have `gz-sim` set this data